src/ec/packages/ec-component-breadcrumb-core/breadcrumb-core.html.twig

```diff
@@ -0,0 +1,126 @@
+{% spaceless %}
+
+{#
+  Parameters:
+    - "links" (array) (default: []): format: [
+      {
+        "label" (string) (default: '') Label of link
+        "path" (string) (default: '') Url of link
+      },
+      ...
+    ],
+    - "navigation_text" (string) (default: ''): Text of navigation in breadcrumb
+    - "ellipsis_label" (string) (default: ''): Label of the ellipsis, e.g. "Click to expand"
+    - "icon_file_path" (string) (default: ''): URL to icons file
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (optional) (string)
+        ...
+      ]
+#}
+
+{# Internal properties #}
+
+{% set _breadcrumb_size = links|length %}
+{% set _navigation_text = navigation_text|default('') %}
+{% set _ellipsis_label = ellipsis_label|default('') %}
+{% set _ellipsis_text = ellipsis_text|default('…') %}
+{% set _breadcrumb_segment_class = 'ecl-breadcrumb-core__segment' %}
+{% set _min_items_left = min_items_left|default(1) %}
+{% set _min_items_right = min_items_right|default(2) %}
+{% set _nav_attributes = [
+  { name: 'aria-label', value: _navigation_text },
+  { name: 'data-ecl-breadcrumb-core', value: 'true' }]
+%}
+{% set _breadcrumb_icon = {
```

Comment:
What about if the user decide to change the icon ?

Reply:
That is not an option, @serotonine :)